src/renderers/shared/reconciler/ReactReconciler.js

```diff
@@ -70,7 +74,10 @@ var ReactReconciler = {
    */
   unmountComponent: function(internalInstance, safely) {
     ReactRef.detachRefs(internalInstance, internalInstance._currentElement);
-    return internalInstance.unmountComponent(safely);
```

Comment:
The return value in all cases seemed to be `undefined` so I’m not passing it back.